rec_to_binaries/trodes_data.py

```diff
@@ -1402,13 +1402,20 @@ def _extract_rec_generic(self, export_cmd, export_dir_ext,
                         external_config_filename = None
 
                     export_call = copy.deepcopy(export_cmd)
+                    
+                    # add switch specific parameters 
+                    export_call.extend(export_args)
+                    
+                    # add sorting mode
+                    export_call += ['-sortingmode', '0']
```

Comment:
Is this just a new parameter for LFPs in trodes 2.0+? I think this can be called by the user in `lfp_export_args` if needed (https://github.com/LorenFrankLab/rec_to_binaries/blob/133f6cbb30b47fa292bce39b1fe729510a7f740d/rec_to_binaries/core.py#L25)